replica/compaction_group.hh

```diff
@@ -58,6 +58,7 @@ class compaction_group {
     // Gates async operations confined to a single group.
     seastar::named_gate _async_gate;
     bool _tombstone_gc_enabled = true;
+    int64_t _sstables_repaired_at = 0;
```

Comment:
How is it defined in Cassandra? As a counter or timestamp?

Note: we have to erase it when importing sstables.

Reply:
It is defined as a timestamp. Yes, when we load and stream sstables into a scylla cluster. We will set the sstables as unrepaired.